src/platform/inprocess/mod.rs

```diff
@@ -358,6 +358,7 @@ impl Deref for OsIpcSharedMemory {
 }
 
 impl OsIpcSharedMemory {
+    /// # Safety
```

Comment:
You will need to fill out the Safety comment here or ask clippy to ignore the error.

@sagudev I think you created this function. Do you mind writing a small explanation for why it is unsafe.

Reply:
Same as in https://docs.rs/ipc-channel/latest/ipc_channel/ipc/struct.IpcSharedMemory.html#method.deref_mut